lib/matterStacks.ts

```diff
@@ -614,6 +620,45 @@ export class MatterStack extends Stack {
         return [matterAuditLoggingBucket, dependencies];
     }
 
+    // Creates the S3 bucket that will hold the CRLs for the Matter PKI.
+    private createMatterCrlBucket(name: string): Bucket {
+        const matterCrlBucket = new Bucket(this, name, {
+            versioned: true,
+            blockPublicAccess: {
```

Comment:
the idea is that any Commissioner on the Internet can find the URL to this bucket in DCL and access it, right? Shouldn't it have readonly public access?